packages/web-component/test-data/ej-fra/ej-fra-malformed.ras

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<read-along>
+    <!--this xml is intentionally modified to simulate someone making a mistake with the expected format of this file-->
```

Comment:
Your comment is confusing me. I open a `<divs>` that I close with `<div>`, so I think that's what you mean I should call malformed or ill-formed, right?

It might be interesting to add an actual invalid test case too, with valid XML that fails to match the DTD. But I can already tell you that Web-Component silently accepts it, and just skips text from any element that does not follow the expected structure.

Reply:
ah, oops! never mind, I didn't read until the end of the file.  We do have an invalid test case already, in ReadAlong-Studio, just not in Web-Component... and actually it's good that Web-Component silently skips unknown elements, I think, as that's expected HTML5 behaviour

Reply:
Good to know silently skipping is what we're supposed to do. I wasn't sure if we should make an issue of it or not, but no, not if it's already doing the right thing.